numpy/typing/tests/data/fail/scalars.pyi

```diff
@@ -62,7 +63,7 @@ np.uint64(value=0)  # E: Unexpected keyword argument
 np.complex128(value=0.0j)  # E: Unexpected keyword argument
 np.str_(value='bob')  # E: No overload variant
 np.bytes_(value=b'test')  # E: No overload variant
-np.void(value=b'test')  # E: Unexpected keyword argument
+np.void(value=b'test')  # E: No overload variant
```

Comment:
Isn't this still an unexpected keyword argument given the `/` in the docstring?

Reply:
Yeah, but due to overloads existing, the error changes.  I would agree that the other error text is probably still better :) (although it might miss the note towards overloads).

Reply:
Agree that the older message is more descriptive. This is a common (minor) problem with overloads, but something that’s also very much an upstream issue.